src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -114,7 +118,12 @@ public void run() {
                         LOGGER.log(Level.WARNING, "Failed to parse the polling log",e);
                         cause = new GitHubPushCause(pushBy);
                     }
-                    if (job.scheduleBuild(cause)) {
+                    ParameterizedJobMixIn scheduler = new ParameterizedJobMixIn() {
```

Comment:
There is a job that can schedule build, why do you need do it through parametrized?